inz-demo/internetnz/Issues/nethuiTicket/description.xml

```diff
@@ -0,0 +1,123 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Demo INZ Nethui Ticket</en>
```

Comment:
Let's call it a Nethui access pass

Reply:
Because we're going to use it for access to many things rather than a ticket to 1 thing